include/deal.II/lac/vector_view.h

```diff
@@ -126,9 +126,11 @@ DEAL_II_NAMESPACE_OPEN
  * in the manual).
  *
  * @author Luca Heltai, 2009
+ *
+ * @deprecated Use ArrayView instead.
  */
 template <typename Number>
-class VectorView : public Vector<Number>
+class DEAL_II_DEPRECATED VectorView : public Vector<Number>
```

Comment:
Oh, is this where it goes? I thought it needs to go to after the closing brace of the class and before the following semicolon. At least that's where we've had it in other cases.

Reply:
same here

Reply:
This is at least the correct position if we want to use `[[deprecated]]` in the future and it also works with `__attribute__((deprecated))`.
I'll switch to the other version regardless.for conformity.